src/contact.js

```diff
@@ -0,0 +1,33 @@
+function createIcon(class1, class2) {
```

Comment:
## Use `ES6` syntax

- [ ] Kindly enforce the usage of arrow functions instead of these `function` keywords in the entire codebase. Thank you.